fvm/evm/stdlib/contract.cdc

```diff
@@ -168,21 +168,29 @@ contract EVM {
 
         /// returns the data that is returned from
         /// the evm for the call. For coa.deploy
-        /// calls it returns the address bytes of the
-        /// newly deployed contract.
+        /// calls it returns the code deployed to
+        /// the address provided in the contractAddress field.
         access(all)
         let data: [UInt8]
 
+        /// returns the newly deployed contract address
+        /// if the transaction caused such a deployment
+        /// otherwise the value is empty.
+        access(all)
+        let deployedContractAddress: [UInt8; 20]
+
         init(
             status: Status,
             errorCode: UInt64,
             gasUsed: UInt64,
-            data: [UInt8]
+            data: [UInt8],
+            contractAddress: [UInt8; 20]
```

Comment:
I'm a bit late to the party here - what is this value for `CadenceOwnedAccount.call` - I'd imagine a `0` array since the field isn't optional?

Reply:
I will make it optional, it's a good suggestion